bin/wavTranscription.md

```diff
@@ -0,0 +1,49 @@
+## Transcribing longer audio clips
+
+The Command and GUI tools perform the same operation. Read a wav file of any duration,
+use the WebRTC Voice Activity Detector (VAD) to split it into smaller chunks and finally
+save a consolidated transcript.
+
+### 0. Pre-requisites
+
+```
+sudo apt-get install virtualenv python3-pyqt5 libcanberra-gtk-module
+virtualenv venv
+source venv/bin/activate
+pip3 install webrtcvad; pip3 install deepspeech; pip3 install tensorflow; pip3 install -r requirements.txt
```

Comment:
why not have a `requirements.txt` and then `pip install -r requirements.txt` ?

Reply:
That'll be even better. Will do it.

Reply:
In that case, it'd probably be better if you namespaced all of this into a separate folder like `bin/vad_transcriber` or a new folder in the root of the repository, like `examples/vad_transcriber`. Then we could also add some other scripts people have developed.

Reply:
@reuben Please correct me if I have understood this wrong..
So, I just move all the files to examples/vad_transcriber right ?

Reply:
@b-ak +1 for moving all files to `examples/vad_transcriber`

Reply:
@lissyx @reuben Need your help with pyqt in virtualenv. I've described the problem here
https://forum.qt.io/topic/95077/pyqt5-in-virtualenv-cannot-mix-incompatible-qt-library-error/2

Reply:
@b-ak If I understand correctly, you're getting a clash and the system's PyQt5 takes precedence?

Reply:
i'd suggest hacking around PYTHONPATH maybe

Reply:
@b-ak Found that? https://stackoverflow.com/questions/40438089/force-virtualenv-pip-to-be-used

Reply:
Yes.. both of the libraries are used interchangeably. When the switch happens the error shows up.
I'll check the pythonpath and give you an update.

Reply:
@lissyx No luck. I even tried creating a qt.conf file. It was mentioned here https://github.com/ContinuumIO/anaconda-issues/issues/32
How do you recommend we proceed now ?

Reply:
I'd guess if it's such a known pain, don't bother, just make sure you explicitely document clearly. People will do their best in general to avoid reading and then complain and spam with issues :)

Reply:
So, everything in virtualenv and Qt in virtualenv if people are ready to fight.

Reply:
@lissyx So, after you replied yesterday I started reverting all qt changes to test one final time before I commit the changes. Guess what, even without any qt5 on my system I was getting the same error (very weird). Synaptic, apt purge nothing seemed to work.

I fixed up a live Ubuntu 18.04 USB today, I tried the pyqt5 in virtualenv on the live disk (two times). **It worked out**. The only plausible explanation I can arrive at is:
1. Do not **_risk_** installing the python3-pyqt5 from the repository (though I have tried this once on the live USB, it didn't cause issues).
2. Other pyqt5 applications maybe causing the error.

I have just now tested the same on **my a fresh new install of Ubuntu 18.04, it seems to be working fine**. I thought I'll push the patch with the virtualenv instructions itself. And over the next couple of days as more people test the environment we could take the final call as to stick with it or use the pyqt5 from the repositories.

Reply:
@b-ak Thanks! If there's some risk, it'd be great you document them, to avoid people get headaches :)